src/components/LogStream/LogsTable.tsx

```diff
@@ -8,6 +8,7 @@ import { FixedSizeList as List } from "react-window";
 import { MultiFilterSelect } from "../Common/MultiFilterSelect";
 import { applyFilters } from "./useFilters";
 import { LogItem } from "./LogItem";
+import {LogsTableHeader} from "./LogsTableHeader";
```

Comment:
It's not a real problem here but why is the spacing different?
It should not be possible, due to prettier autoformatting and everything.
We should investigate.